kubemarine/procedures/install.py

```diff
@@ -52,14 +52,29 @@ def system_prepare_check_system(cluster):
             raise Exception('%s host operating system is unsupported' % address)
         if context["os"]["family"] == "unknown":
             supported_os_versions = []
-            os_family_list = cluster.globals["compatibility_map"]["distributives"][context["os"]["name"]]
+            os_family_list = cluster.globals["compatibility_map"]["distributives"]
             for os_family_item in os_family_list:
-                supported_os_versions.extend(os_family_item["versions"])
-            raise Exception("%s running on unknown %s version. "
-                            "Expected %s, got '%s'" % (address,
-                                                       context["os"]["name"],
-                                                       supported_os_versions,
-                                                       context["os"]["version"]))
+                supported_os_versions.append(os_family_item)
+            if cluster.context['execution_arguments']['os_family'] in supported_os_versions:
```

Comment:
When I suggested --os-family option, I though it will make KubeMarine act as the target nodes have the specified family. But in fact this parameter is used only to check that the specified os_family is among supported (`in supported_os_versions`).